tiny_dnn/core/kernels/deconv2d_grad_op.h

```diff
@@ -0,0 +1,49 @@
+/*
+    Copyright (c) 2017, Taiga Nomi
+    All rights reserved.
+
+    Use of this source code is governed by a BSD-style license that can be found
+    in the LICENSE file.
+*/
+#pragma once
+
+#include "tiny_dnn/core/framework/op_kernel.h"
+#include "tiny_dnn/core/kernels/deconv2d_op_internal.h"
+
+namespace tiny_dnn {
+
+class Deconv2dGradOp : public core::OpKernel {
```

Comment:
better name for this: `Conv2dTranposed`

see: https://github.com/tiny-dnn/tiny-dnn/pull/786#discussion_r125084038